doc/modules/preprocessing.rst

```diff
@@ -310,10 +310,103 @@ The normalizer instance can then be used on sample vectors as any transformer::
   efficient Cython routines. To avoid unnecessary memory copies, it is
   recommended to choose the CSR representation upstream.
 
-.. _preprocessing_binarization:
```

Comment:
On the master branch, I did `grep preprocessing_binarization --include="*.rst" . -nr`, and found that this was the only instance where this header was being called.